packages/edit-post/src/components/sidebar/plugin-document-setting/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Defines as extensibility slot for the Settings sidebar
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill, PanelBody } from '@wordpress/components';
+
+export const { Fill, Slot } = createSlotFill( 'PluginSettingsSidebar' );
+
+/**
+ * Renders items below the Status & Availability panel in the Document Sidebar.
+ *
+ * @param {Object} props Component properties.
+ * @param {string} [props.className] An optional class name added to the row.
+ *
+ * @example <caption>ES5</caption>
+ * ```js
+ * // Using ES5 syntax
+ * var el = wp.element.createElement;
+ * var __ = wp.i18n.__;
+ * var registerPlugin = wp.plugins.registerPlugin;
+ * var PluginDocumentSetting = wp.editPost.PluginPostStatusInfo;
+ * function MyDocumentSettingPlugin() {
+ * 	return el(
+ * 		PluginDocumentSetting,
+ *		{
+ * 			className: 'my-document-setting-plugin',
+ * 		},
+ * 		__( 'My Document Setting Panel' )
+ * 		);
+ * 	}
```

Comment:
Nit: missing semicolon:
```suggestion
 * 	};
```